js/lang/fr.js

```diff
@@ -0,0 +1,47 @@
+var app = app || {};
+
+app.lang = app.lang || {};
+
+app.lang['fr'] = (function() {
+
+	return {
+		'language.cs': 'Czech',
+		'language.en': 'English',
+		'language.es': 'Spanish',
+    'language.fr': 'French',
```

Comment:
Where are the accents?
Please, use more formal language: e.g. not "y a pas de réseau", but "il n'y a pas de réseau".

Reply:
ok I will. thanks